crates/lib3h_protocol/src/lib.rs

```diff
@@ -1,5 +1,10 @@
 //! This module provides the api definition for working with lib3h
 
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+extern crate rmp_serde as rmps;
```

Comment:
I find it easier to figure out what's going on with original crate names, but I'm totally cool if others feel differently.